recipes/simuPOP/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "simuPOP" %}
+{% set version = "1.1.8.3" %}
+{% set md5 = "677d2677b0c10bee1e9f62cb6a85969e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  skip: True  # [py27]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - zlib 1.2.*
+    - swig
+  run:
+    - python
+    - zlib 1.2.*
+
+test:
+  imports:
+    - simuPOP
+    - simuOpt
+
+about:
+  home: http://github.com/BoPeng/simuPOP
+  license: GPL3
```

Comment:
I prefer `GPL-3.0`, but this is minor and can be changed in the feedstock later.